Annotations/Classes/TextAnnotation/MouseEventsHandler.swift

```diff
@@ -76,7 +76,8 @@ class MouseEventsHandler {
 
     switch textContainerView.transformState {
     case .scale:
-      textContainerView.reduceWidthIfNeeded()
+      // FIXME: - Ensure that width is reduced correctly and doesn't affect text resizing
+      /* textContainerView.reduceWidthIfNeeded() */
```

Comment:
remove commented code

Reply:
@memstel this was left intentionally to not forget the idea and should be fixed later.